config/bundlesize.json

```diff
@@ -0,0 +1,8 @@
+{
+    "files": [
+        {
+            "path": "./dist/fundamental-styles.css",
+            "maxSize": "70 kB"
```

Comment:
This is the file that would need to be changed as the bundle grows - it's currently at ~68kb.